packages/client/src/api/constants.ts

```diff
@@ -0,0 +1,9 @@
+const TEAM_NAME = 'lunatics';
+const RATING_FIELD_NAME = `${TEAM_NAME}__score`;
+const LEADER_LIMIT_USERS = 10;
+
+export {
+	TEAM_NAME,
+	RATING_FIELD_NAME,
+	LEADER_LIMIT_USERS,
+};
```

Comment:
Но, думаю читабельнее будет если:

```suggestion
export const TEAM_NAME = 'lunatics';
export RATING_FIELD_NAME = `${TEAM_NAME}__score`;
export LEADER_LIMIT_USERS = 10;
```